docs/estela/installation/main-guide.md

```diff
@@ -19,16 +19,24 @@ architectures (soon).
 
 ## Requirements
 
-- Docker v20.10.x with *docker-compose*
-- Kubectl >= v1.23.x
-- Helm >= v3.9.x
-- yarn v1.22.x
-- node v14.x
+We advise you to use Linux based system to run all the commands, but you can also use Windows Subsystem for Linux ([WSL](https://learn.microsoft.com/en-us/windows/wsl/install){:target="_blank"}) on windows systems.
+- Docker v20.10.x with *docker-compose*.  
+*Note:* Is recommended to use docker-engine instead of docker-desktop for performance purposes check the following resources for installation: [docker-engine installation](https://docs.docker.com/engine/install/){:target="_blank"}, [docker-desktop installation](https://docs.docker.com/desktop/){:target="_blank"}.
+- Kubectl >= v1.23.x  
+*Note:* Kubectl comes with Minikube so is not mandatory to install it apart of Minikube, also to avoid incompatibilities, to learn more check this resource: [minikube+kubectl installation](https://minikube.sigs.k8s.io/docs/start/){:target="_blank"}.
```

Comment:
The installation of kubectl is mandatory since minikube is an optional requirement. Please remove this Note.